README.md

```diff
@@ -1 +1,126 @@
-# OOP-School-library
\ No newline at end of file
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Decoding\_Morse\_Code ](#-decoding_morse_code-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+  - [Setup](#setup)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Decoding_Morse_Code <a name="about-project"></a>
+
+The purpose of this app is to create a school library using Ruby as the programming language.
+
+## 🛠 Built With <a name="built-with"></a>
+
+The project is built to create a school library using OOP rules of Ruby 
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+The project is built using the following technologies:
+
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Class**
+- **Inheritance**
+- **Polymorphism**
+- **Gems install**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+## Setup
+
+To get a local copy up and running, follow these steps.
+
+1. Clone this repository to your local machine:
+
+```
+git clone https://github.com/ZabihullahNooriWardak/OOP-School-library.git
+
+```
+
```

Comment:
- [ **OPTIONAL** ] What do you think if you improve your `Getting Started` section of your README with other information needed by users in order to use your project locally(such as how to `access` the cloned dir, `use or run` the app, run tests if exixst,...) To help you, I recommend you use this **[Gist_file](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)** as template.